README.md

```diff
@@ -217,6 +217,7 @@ the specified context and less than the context for the next release.
 | jwst tag            | DMS build | SDP_VER  | CRDS_CONTEXT | Released   | Ops Install | Notes                                         |
 |---------------------|-----------|----------|--------------|------------|-------------|-----------------------------------------------|
 | 1.15.0              | B11.0rc1  | TBD      | 1241         | 2024-06-26 | TBD         | First release candidate for B11.0             |
+| 1.14.1              |           |          | 1215         | 2024-06-27 |             | PyPI-only release for external users          |
```

Comment:
I think we can leave the CRDS entry for this release blank, because it's not an official build. Or at least set it to the same value as 1.14.0 (i.e. 1238).

Reply:
```suggestion
| 1.14.1              |           |          | 1238         | 2024-06-27 |             | PyPI-only release for external users          |
```